SPECS-EXTENDED/CharLS/CharLS.spec

```
@@ -0,0 +1,164 @@
+Summary:        An optimized implementation of the JPEG-LS standard
+Name:           CharLS
+Version:        2.0.0
+Release:        10%{?dist}
+License:        BSD
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://github.com/team-charls/charls
+Source0:        https://github.com/team-charls/charls/archive/%{version}/%{name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```

It already has `name-version.ext` format.
```suggestion
Source0:        https://github.com/team-charls/charls/archive/%{version}/%{name}-%{version}.tar.gz
```